packages/medusa/src/services/cart.ts

```diff
@@ -829,13 +888,15 @@ class CartService extends BaseService {
    * @return the result of the update operation
    */
   async updateCustomerId_(cart: Cart, customerId: string): Promise<void> {
-    const customer = await this.customerService_
-      .withTransaction(this.transactionManager_)
-      .retrieve(customerId)
-
-    cart.customer = customer
-    cart.customer_id = customer.id
-    cart.email = customer.email
+    return this.atomicPhase_(async (transactionManager: EntityManager) => {
```

Comment:
sorry @srindom but for me that method is not private since by default a class member is public unless we specify that it is private :) 
I can't not refer to the name as well because it is container `_`. But event `atomicPhase_` contains the `_` but this member is either public or protected otherwise it would not be accessible from the derivates class

Reply:
How do I identify a private method agains public/protected?

Reply:
yes - exactly; the suffix indicates that it is private. 

I am aware that it is not technically private, but using the `_` consumers are at least aware that this method is not supposed to be called from outside of the method.

Reply:
The convention is inspired by Google: https://google.github.io/styleguide/jsguide.html#naming-method-names

Reply:
We may want to revise this, however, given the migration to TS